src/Core/AspectKernel.php

```diff
@@ -58,6 +59,13 @@
     protected static $containerClass = 'Go\Core\GoAspectContainer';
 
     /**
+     * Default class name for cache path resolver, can be redefined in children
+     *
+     * @var string
+     */
+    protected static $cachePathResolverClass = 'Go\Instrument\ClassLoading\CachePathResolver';
```

Comment:
Ok, thank you,
I was thinking of doing so, but found this way of setting up container itself, so decided to mirror it.